### PR TITLE
Add "support us" section

### PR DESCRIPTION
**Pull request by thoeni**
I'd like to introduce the concept of "supporters" alongside "sponsors". Supporters instead of helping us financially, will give us licenses or products they directly make (i.e. Jetbrains, 1Password, Stickermule...)

### CODE REVIEW

**Comment by domgreen** [comment]
As @myitcv mentioned in chat as the sponsors make such a great contribution we need to make sure that is represented when we talk and promote those people.
If we are adding the supporters in we should just keep it to this section / smaller logo especially if we have a number of recurring sponsors.

File: sponsorship/README.md [hunk 1]
```diff
@@ -44,6 +44,18 @@ In addition, recurring sponsors will be listed prominently on our website.
 
 Because we are a non-profit organisation, we will typically put any surplus funds towards conference scholarship funds such as https://www.gophercon.co.uk/scholarships/.
 
+## 👨‍👩‍👦‍👦 Support us
+
+If your business is about products or services that could improve the overall meetup experience, and benefit our community, please consider supporting this group by providing us community licenses or access to free merchandise! Great examples of companies that have already agreed on this kind of "sponsorship" are:
+
+* Jetbrains: three free licenses every month for our attendees to give away in a raffle fashion
```
Comment (katzien) [inline sponsorship/README.md]:
```suggestion
* Jetbrains: three free licenses every month for our attendees to give away in a raffle
```

File: sponsorship/README.md [hunk 2]
```diff
@@ -44,6 +44,18 @@ In addition, recurring sponsors will be listed prominently on our website.
...
+* 1Password: they granted us a community account that we can use to share admin/passwords/details within us organisers
```
Comment (katzien) [inline sponsorship/README.md]:
```suggestion
* 1Password: a free community license for the organisers to be able to share passwords and sensitive data securely
```

File: sponsorship/README.md [hunk 3]
```diff
@@ -44,6 +44,18 @@ In addition, recurring sponsors will be listed prominently on our website.
...
+* Stickermule: they kindly provide stickers for our events
```
Comment (katzien) [inline sponsorship/README.md]:
```suggestion
* Stickermule: stickers with the London Gophers logo to be distributed at our events
```

File: sponsorship/README.md [hunk 4]
```diff
@@ -44,6 +44,18 @@ In addition, recurring sponsors will be listed prominently on our website.
 
 Because we are a non-profit organisation, we will typically put any surplus funds towards conference scholarship funds such as https://www.gophercon.co.uk/scholarships/.
 
+## 👨‍👩‍👦‍👦 Support us
+
+If your business is about products or services that could improve the overall meetup experience, and benefit our community, please consider supporting this group by providing us community licenses or access to free merchandise! Great examples of companies that have already agreed on this kind of "sponsorship" are:
```
Comment (katzien) [inline sponsorship/README.md]:
```suggestion
If your business is about products or services that could improve the overall meetup experience, and benefit our community, please consider supporting this group by providing us community licenses or access to free merchandise! Great examples of companies which have supported us this way are:
```

File: sponsorship/README.md [hunk 5]
```diff
@@ -44,6 +44,18 @@ In addition, recurring sponsors will be listed prominently on our website.
...
+
+* Jetbrains: three free licenses every month for our attendees to give away in a raffle fashion
+* 1Password: they granted us a community account that we can use to share admin/passwords/details within us organisers
+* Stickermule: they kindly provide stickers for our events
+
+We're incredibly grateful to everyone who would like to contribute, one way or another, and we'll make sure our friends will know about your kindness!
```
Comment (katzien) [inline sponsorship/README.md]: I don't think this paragraph is necessary, or adds much to what has already been said? Just trying to make sure this page doesn't get too long... :)

File: sponsorship/README.md [hunk 6]
```diff
@@ -44,6 +44,18 @@ In addition, recurring sponsors will be listed prominently on our website.
...
+
+---
+
 As mentioned above, the organising committee of London Gophers are volunteers. We are always on the lookout for people keen to join our team!
```
Comment (katzien) [inline sponsorship/README.md]: Shouldn't this go with the previous section?

File: sponsorship/README.md (hunk 5 above)
Comment (thoeni) [reply sponsorship/README.md]: Good point, we want to keep it short and sweet, happy to remove this last paragraph 👍

File: sponsorship/README.md (hunk 6 above)
Comment (thoeni) [reply sponsorship/README.md]: I was thinking of having:
- Sponsorship
- Support
-> Closing

This bit doesn't look specific to either one, and putting it above sounds like it's related to sponsorship _and then_ we talk about supporters... that was a conscious decision, but if it's not clear it should be revisited 🤔

**Pull request merged**